components/RPC-Connection.jsx

```diff
@@ -172,6 +199,9 @@ function applyFilter(value, filter, network, setReturnValue) {
 		case "blocksToDays":
 			value = (value * 6) / 86400;
 			break;
+		case "lowStaked":
+			value = (value * 6) / 86400;
+			break;
```

Comment:
I don't think this filter is being used?

Reply:
Yeah, clearly I did not delete the unused code :)